libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -336,7 +336,7 @@ private bool EvalExpressionInCondition(string exp)
             }
         }
 
-        private object EvalExpression(string exp)
+        private object EvalExpression(string exp, bool nullable = true)
```

Comment:
>nullable [](start = 55, length = 8)

throwOnNull?

Reply:
Do you mean the variable's name? or its meaning?
When `nullable` is true, we allow the expression to be evaluated to null, and the result will be a string "null" instead.

Reply:
yes, i think Tom mean that :)